<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simulador</class>
 <widget class="QDialog" name="simulador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>SIMULADOR</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>541</width>
     <height>351</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="icon">
     <iconset>
      <normaloff>../iconos/user_friend.png</normaloff>../iconos/user_friend.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Usuario</string>
    </attribute>
    <widget class="QPushButton" name="login_usuario">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingresar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../iconos/user_friend.png</normaloff>../iconos/user_friend.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PIN</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Numero de tarjeta</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tarjeta_simulador">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>251</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="pin_simulador">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>251</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="productos">
    <attribute name="icon">
     <iconset>
      <normaloff>../iconos/apple-red.png</normaloff>../iconos/apple-red.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Agregar productos</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo de producto</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Marca</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Precio</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nombre_producto">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="marca_producto">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="precio_producto">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ingresar_producto">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>220</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingresar producto</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tipo_producto">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="icon">
     <iconset>
      <normaloff>../iconos/money_91337.jpg</normaloff>../iconos/money_91337.jpg</iconset>
    </attribute>
    <attribute name="title">
     <string>Administracion de productos</string>
    </attribute>
    <widget class="QTableWidget" name="tabla_productos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>461</width>
       <height>192</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>153</number>
     </attribute>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tipo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Marca</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="cargar_productos">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Cargar datos</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../iconos/base.png</normaloff>../iconos/base.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="ver_producto">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>230</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ver producto</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../iconos/search_button_green_32.png</normaloff>../iconos/search_button_green_32.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
